packages/runner/src/worker_inner.rs

```diff
@@ -44,16 +44,20 @@ pub struct ControllerCfg {
     pub session: u64,
     pub auth: Arc<dyn Authorization>,
     pub handshake: Arc<dyn HandshakeBuilder>,
+    // ASK: Why is this never used?
+    // This is redefined in SdnInnerCfg below
     pub tick_ms: u64,
     #[cfg(feature = "vpn")]
     pub vpn_tun_device: Option<sans_io_runtime::backend::tun::TunDevice>,
 }
 
 pub struct SdnInnerCfg<SC, SE, TC, TW> {
     pub node_id: NodeId,
+    // ASK: Maybe we should somehow use controller.unwrap().tick_ms?
```

Comment:
`tick_ms` from the above struct is never used. I am wondering if we should one of them.